play.lic

```diff
@@ -0,0 +1,43 @@
+custom_require.call %w[events]
+class Play
+  def initialize
+    @song ||= UserVars.song
+    check_flags
+    main_loop 
+  end
+
+  def clean?
+    return !Flags['clean']
+  end
+
+  def check_flags
```

Comment:
Dont bother. Let other scripts clean instruments. Dirty instruments are, anecdotally, not impacting performance training.